x-pack/functionbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+[id="{beatname_lc}-getting-started"]
+== Getting Started With {beatname_uc}
+
+include::{libbeat-dir}/docs/shared-getting-started-intro.asciidoc[]
+
+* <<{beatname_lc}-installation>>
+* <<{beatname_lc}-configuration>>
+* <<{beatname_lc}-template>>
+* <<{beatname_lc}-deploying>>
+* <<view-kibana-dashboards>>
+* <<setup-repositories>>
+
+[id="{beatname_lc}-installation"]
+=== Step 1: Download the {beatname_uc} package
+
+The {beatname_uc} package contains the command line tools, configuration file,
+and binary code required to run {beatname_uc} in your serverless environment. 
+
+To download and extract the package, use the commands that work with your
+system.
+
+// REVIEWERS: will the download path include x-path? In BC2, it's 
+// https://staging.elastic.co/6.5.0-92a7b7db/downloads/beats/x-pack/functionbeat/functionbeat-6.5.0-windows-x86_64.zip
```

Comment:
I believe it will have it since a snapshot is promoted to GA. cc @tsg

Reply:
The path shown on the download page in staging does not show x-path: https://staging-website.elastic.co/downloads/beats/functionbeat